lua/tardis-nvim/init.lua

```diff
@@ -18,27 +18,22 @@ local function get_git_root()
     return Job:new({
         command = 'git',
         args = { 'rev-parse', '--show-toplevel' },
-        on_exit = function (j, status)
-            if status ~= 0 then
-                error("git couldn't resolve the root", 4)
-            end
-
-            return j:result()
-        end
     }):sync()[1]
```

Comment:
Won't this fail when the table is empty? I might be missing something.

Reply:
It returns `nil` from my testing. I've moved the error check to the `tardis` function further down